actors/miner/src/types.rs

```diff
@@ -135,6 +135,34 @@ pub struct ProveCommitSectorParams {
     pub proof: RawBytes,
 }
 
+// Note no UnsealedCID because it must be "zero" data.
+#[derive(Clone, Debug, Eq, PartialEq, Serialize_tuple, Deserialize_tuple)]
+pub struct SectorNIActivationInfo {
```

Comment:
This has changed in the FIP since you wrote this. Add sealing_number separate from sector_number.